src/lightning_app/runners/cloud.py

```diff
@@ -571,24 +663,27 @@ def _get_run_body(
         """Get the specification of the run creation request."""
         # The entry point file needs to be relative to the root of the uploaded source file directory,
         # because the backend will invoke the lightning commands relative said source directory
-        app_entrypoint_file = Path(self.entrypoint_file).absolute().relative_to(root)
+        app_entrypoint_file = Path(self.entrypoint).absolute().relative_to(root)
 
         run_body = CloudspaceIdRunsBody(
             cluster_id=cluster_id,
             app_entrypoint_file=str(app_entrypoint_file),
```

Comment:
If a directory is passed, `app_entrypoint_file` won't be runnable. Might be better to have it empty or not even create a Run then.

Reply:
Good callout, unfortunately the backend gives an error if the entrypoint isn't provided so we have to keep it this way for now :/

Reply:
Mind adding a comment ?